planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -1415,6 +1415,39 @@ void StartPlannerModule::setDebugData()
     add(start_pose_text_marker_array);
   }
 
+  // visualize shift start pose to end pose footprints
```

Comment:
nits: not just shift but geometirc

Reply:
I used pull_out_star/end_pose for the variable names in module side

Reply:
how about this?
  // visualize the footprint from pull_out_start pose to pull_out_end pose along the path